### ISSUE TITLE
Change App's Name to "Nextcloud"

### ISSUE THREAD

**Issue by MariusBluem** [nextcloud/windows-universal#19]
We should consider about renaming the application to "Nextcloud". I think the "App" is not needed there, because everybody shopping in Windows Store does know that he is searching for apps 😜 ... this would also makes the whole thing more "official" ;)

**Nextcloud App** -> **Nextcloud**

@SunboX

**Comment by SunboX** [comment]
Sadly, the name "Nextcloud" is already taken by another person. I don't know by whom. So I decided to name it "Nextcloud App". Seemed the best fit to me.

**Comment by MariusBluem** [comment]
Okay... The Nextcloud App here is the only one I find when I am searching ... Mind talking with Microsoft's Support? 😁

**Comment by SunboX** [comment]
You can reserve app names without actual publishing a app at the Microsoft Store. But you are right, I will contact Microsoft and ask.

**Comment by SunboX** [comment]
Just for reference, a note from Microsoft:

> **Note** Because names can be reserved for one year, you might find that you can't reserve a name, even though you don't see any apps listed by that name in the Store. This is usually because another developer has reserved the name for their app but hasn't submitted it yet. If you are unable to reserve a name for which you hold the trademark or other legal right, or if you see another app in the Windows Store using that name, contact Microsoft.
[https://msdn.microsoft.com/en-us/windows/uwp/publish/create-your-app-by-reserving-a-name]

As I said, I will contact Microsoft. 😏

**Comment by MariusBluem** [comment]
@karlitschek may help out, because of the Nextcloud Trademark 🤔

**Comment by karlitschek** [comment]
Sure. This app can be called Nextcloud or Nextcloud App as long as it doesn't harm the Nextcloud project. See our trademark guidelines for more information :-)  https://nextcloud.com/trademarks/

**Comment by MariusBluem** [comment]
That's not exactly what I've meant. It seems that somebody have registred "Nextcloud" as Application name for the Microsoft Store ... According to the fact, that this will get our Nextcloud app for Windows Mobil (and UWP) we should take care about claiming this name 😉 ...lets see what @SunboX reports from the Microsoft guys, but maybe we need some letter from the Nextcloud GmbH to Microsoft, that we want to claim this name there because of the trademark Nextcloud GmbH is holding 😁 @karlitschek

**Comment by SunboX** [comment]
@karlitschek 

I wanted to report the trademark infringement issue to Microsoft (someone else reserved the app name). But for that I need the trademark registration number (see below). Can you tell it to me?

> All trademark submissions **must** include a trademark registration number and the country in which the registration was granted.

**Comment by DecaTec** [comment]
@SunboX 
The name Nextcloud was taken by me, because I planned making an app.
But I'm totally ok with giving the name free for this project. I'm not at home atm, but I'll contact you later so we can talk about the details.

**Comment by SunboX** [comment]
Hey @DecaTec, that's really great! Thanks for being so social! :smile: I'm not at home, too, but I can reserve the name from my mobile. Let me know if you have released it. 

If you like, you can contribute to this project. Would really be great!

**Comment by SunboX** [comment]
I successfully reserved the "Nextcloud" name, thanks to @DecaTec !

The app with the new name was uploaded to the Store as v1.0.3 some minutes ago and is now waiting for certification. This will take one more day and one for the publishment. I've also included some minor bug-fixes and enhancements. See release https://github.com/nextcloud/windows-universal/releases/tag/1.0.3

I will inform you and close this issue as soon as it is published.
